### PR TITLE
CGT-1220 will need new acceptance tests before merging.

### PR DESCRIPTION
**Pull request by DavidWoodward37**

### CODE REVIEW

**Comment by MJCallahanPage** [comment]
Waiting for acceptance tests changes

**Comment by jameshforster** [comment]
Acceptance test changes added CGT-1203

**Comment by MJCallahanPage** [comment]
Reviewing now

File: app/constructors/NoTaxDateConstructor.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import models.DisposalDateModel
+
+object NoTaxDateConstructor extends NoTaxDateConstructor
+
+trait NoTaxDateConstructor {
+
+  def buildDate(data: DisposalDateModel): String = {
+    Dates.datePageFormat.format(Dates.constructDate(data.day, data.month, data.year))
+  }
+}
```
Comment (MJCallahanPage) [inline app/constructors/NoTaxDateConstructor.scala]: Can you delete this please as it is no longer used. Thanks :smile: :banana:

**Pull request merged**